connector_jira_tempo/models/jira_backend/common.py

```diff
@@ -88,15 +88,24 @@ def _update_ts_line_status(self, date_from, date_to, state, user_ids):
         })
         self._validate_ts(date_from, date_to, state, user_ids)
 
-    def _get_ts_lines(self, date_from, date_to, user_ids):
+    def _get_ts_lines(self, date_from, date_to, user_ids,
+                      only_non_validated=False):
         ts_line_model = self.env['account.analytic.line']
-        return ts_line_model.search([
+        domain = [
             # TODO: any better filter here?
             # `is_timesheet` is not available since we don't use ts_grid
+            # But `is_timesheet` is a computed field with value:
+            # project_id setted
+            ('project_id', '!=', False),
             ('date', '>=', date_from),
             ('date', '<=', date_to),
             ('user_id', 'in', user_ids)
-        ])
+        ]
+        if only_non_validated:
```

Comment:
I haven't found usage of this one anywhere

Reply:
it's optional, we use it in our custom code

Reply:
Got it, yet shouldn't then it be in that custom module as overriden? even `validated` field is not defined anywhere. Seems that this method should've been extended in that custom code? Via refactor as "_get_ts_line_domain"?

Reply:
oh wait, you are right... this field comes from enterprise.

@jcoux we can do better here: instead of passing a flag:

1. remove the flag param
2. split domain building to `_get_ts_lines_domain`
3. retrieve domain

then anyone can override domain generation based on whatever param you'd like to pass around via ctx key.

@alexey-pelykh good catch, tnx :)

Reply:
Np :) Also, please take a look at https://github.com/OCA/connector-jira/blob/ddf8e1560511e3197c0da373afbdd63df89a3056/connector_jira/models/account_analytic_line/importer.py#L32 seems that if this field is computed and we set `project_id`, this block is not really needed

@jcoux if possible please do a PR for v12 as well since I've already forward-ported this one :D

Reply:
@alexey-pelykh @simahawk Fix is here:
https://github.com/OCA/connector-jira/pull/40